utils/testrunner/test/testrunner.spec.js

```diff
@@ -855,3 +1011,63 @@ module.exports.addTests = function({describe, fdescribe, xdescribe, it, xit, fit
   });
 };
 
+class TestTracer {
```

Comment:
We have some other tests that just trace hooks, perhaps we should migrate them to TestTracer in a follow up?